Source/common/SNTStoredEvent.m

```diff
@@ -70,6 +70,7 @@ - (void)encodeWithCoder:(NSCoder *)coder {
   ENCODE(self.quarantineRefererURL, @"quarantineRefererURL");
   ENCODE(self.quarantineTimestamp, @"quarantineTimestamp");
   ENCODE(self.quarantineAgentBundleID, @"quarantineAgentBundleID");
+  ENCODE([NSNumber numberWithBool:self.standaloneApproval], @"standaloneApproval");
```

Comment:
```suggestion
  ENCODE(@(self.standaloneApproval), @"standaloneApproval");
```

Reply:
This should be entirely removed, as this field is no longer required after we move to the audit event.